LICENSE

```diff
@@ -1,4 +1,5 @@
 Copyright 2021 Assanali Mukhanov & Top.gg
+Copyright 2024 null & Top.gg
```

Comment:
I don't think "null" is legally binding 😋

Reply:
Oh sorry

Reply:
Should i just change it to null8626?

Reply:
I assume `@null8626` would work? Copyright is complicated.

Reply:
Alright!

Reply:
Done! A pull request in the Rust SDK has been created related to this change. https://github.com/Top-gg-Community/rust-sdk/pull/20